<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>27</red>
        <green>27</green>
        <blue>27</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>40</red>
        <green>40</green>
        <blue>40</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>33</red>
        <green>33</green>
        <blue>33</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>13</red>
        <green>13</green>
        <blue>13</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>18</red>
        <green>18</green>
        <blue>18</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>27</red>
        <green>27</green>
        <blue>27</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>13</red>
        <green>13</green>
        <blue>13</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="127">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Accent">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>27</red>
        <green>27</green>
        <blue>27</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>40</red>
        <green>40</green>
        <blue>40</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>33</red>
        <green>33</green>
        <blue>33</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>13</red>
        <green>13</green>
        <blue>13</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>18</red>
        <green>18</green>
        <blue>18</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>27</red>
        <green>27</green>
        <blue>27</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>13</red>
        <green>13</green>
        <blue>13</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="127">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Accent">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>13</red>
        <green>13</green>
        <blue>13</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>27</red>
        <green>27</green>
        <blue>27</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>40</red>
        <green>40</green>
        <blue>40</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>33</red>
        <green>33</green>
        <blue>33</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>13</red>
        <green>13</green>
        <blue>13</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>18</red>
        <green>18</green>
        <blue>18</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>13</red>
        <green>13</green>
        <blue>13</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>13</red>
        <green>13</green>
        <blue>13</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>27</red>
        <green>27</green>
        <blue>27</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>27</red>
        <green>27</green>
        <blue>27</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>27</red>
        <green>27</green>
        <blue>27</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="127">
        <red>13</red>
        <green>13</green>
        <blue>13</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Accent">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>19</red>
        <green>19</green>
        <blue>19</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>20</y>
      <width>1000</width>
      <height>560</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>203</red>
          <green>203</green>
          <blue>203</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Mid">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>26</red>
          <green>26</green>
          <blue>26</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>39</red>
          <green>39</green>
          <blue>39</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>203</red>
          <green>203</green>
          <blue>203</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Mid">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>26</red>
          <green>26</green>
          <blue>26</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>39</red>
          <green>39</green>
          <blue>39</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>203</red>
          <green>203</green>
          <blue>203</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Mid">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>26</red>
          <green>26</green>
          <blue>26</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="Main_tab">
     <attribute name="title">
      <string>Main</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="Connection_tab">
     <attribute name="title">
      <string>Connection</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="Analysis_tab">
     <attribute name="title">
      <string>Analysis</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="Settings_tab">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QComboBox" name="LanguageBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <hintingpreference>PreferDefaultHinting</hintingpreference>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="currentText">
       <string> Polski</string>
      </property>
      <item>
       <property name="text">
        <string> Polski</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string> English</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="LanguageLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Language</string>
      </property>
     </widget>
     <widget class="QLabel" name="EngineLimitLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Engine Limit</string>
      </property>
     </widget>
     <widget class="QComboBox" name="EngineLimitBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>110</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string> 40</string>
      </property>
      <item>
       <property name="text">
        <string> 40</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string> 50</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string> 60</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string> 70</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string> 80</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string> 90</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string> Off</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="IPBox">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>60</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentText">
       <string>192.168.0.100</string>
      </property>
      <item>
       <property name="text">
        <string>192.168.0.100</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>192.168.0.101</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>192.168.0.102</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>192.168.0.103</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>192.168.0.104</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>192.168.0.105</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>192.168.0.106</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>192.168.0.107</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>192.168.0.108</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>192.168.0.109</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>192.168.0.110</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="IPLabel">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>60</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>IP</string>
      </property>
     </widget>
     <widget class="QLabel" name="PortLabel">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>110</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
     <widget class="QComboBox" name="PortBox">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>110</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentText">
       <string>12345</string>
      </property>
      <item>
       <property name="text">
        <string>12345</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12346</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12347</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="SaveSettingsButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>430</y>
        <width>171</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DefaultSettingsButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>430</y>
        <width>171</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Default</string>
      </property>
     </widget>
     <widget class="QLabel" name="ThemeLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Theme</string>
      </property>
     </widget>
     <widget class="QComboBox" name="ThemeBoox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string>Dark</string>
      </property>
      <item>
       <property name="text">
        <string>Dark</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="FontLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Font</string>
      </property>
     </widget>
     <widget class="QComboBox" name="FontBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>210</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string>Segoe UI</string>
      </property>
      <item>
       <property name="text">
        <string>Segoe UI</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="FontSizeLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Font Size</string>
      </property>
     </widget>
     <widget class="QComboBox" name="FontSizeBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>260</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string>12</string>
      </property>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QToolButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>10</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset theme="application-exit"/>
    </property>
    <property name="iconSize">
     <size>
      <width>14</width>
      <height>14</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
